AUTHORS

```diff
@@ -233,7 +233,6 @@ Prasoon Shukla <prasoon92.iitr@gmail.com>
 Stefen Yin <zqyin@ucdavis.edu>
 Thomas Hisch <t.hisch@gmail.com>
 Madeleine Ball <mpball@gmail.com>
-Case Van Horsen <casevh@gmail.com>
```

Comment:
I don't know why this name was removed. I ran the script locally and it didn't remove this name or the ones below that were removed.